.github/workflows/integration.yaml

```diff
@@ -48,4 +48,4 @@ jobs:
         key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
     - name: Run tests
       run: './run_tests.sh -integration test/integration -- -k "${{ matrix.subset }}"'
-      working-directory: 'galaxy root'
+      working-directory: 'galaxy-root'
```

Comment:
You can just drop this line and line 39 to avoid the space. But we do need to have this fixed before merging.

Reply:
I re-named them to galaxy in the subsequent push. When we have a fix for the bug (space in dir name), I can change it back to verify.

Reply:
Removed line 39 and 51 locally.